recipes/Media/ASR/seq2seq/train_with_wav2vec.py

```diff
@@ -0,0 +1,364 @@
+#!/usr/bin/env python3
+
+# ################################
+# Authors modifier: Gaelle Laperriere
```

Comment:
remove modifier

Reply:
Took it from another recipe. Correction done.